docs/GettingStarted.md

```diff
@@ -252,12 +252,39 @@ Other useful options include:
 4.	`Test_name` to run a single test
 
 ### xdp_tests.exe
-This application tests various XDP functionalities. It has the following tests:
-1. Reflection Test: This tests the XDP_TX functionality. The following steps show how to run the test:
-   1. On the system under test, install eBPF binaries (`.\scripts\setup-ebpf.ps1`).
-   2. Load the test eBPF program by running the following commands: `netsh`, `ebpf`, `add program reflect_packet.o xdp` and note the ID.
-   3. From a remote host, run xdp_tests.exe and in `--remote-ip` parameter pass an IPv4 or IPv6 address of an Ethernet-like interface on the system under test in string format.
-   4. Unload the program from system under test by running `delete program <id>` on the netsh prompt, where <id> is the ID noted above.
+This application tests various XDP functionalities. These tests require two hosts to run. There are three variations of the XDP tests.
+
+#### Reflection Test
+This tests the XDP_TX functionality.
+1. On the first host:
+   1. Install eBPF binaries (`.\scripts\setup-ebpf.ps1`).
+   2. Load the test eBPF program by running the following command: `netsh ebpf add program reflect_packet.o xdp` and note the ID.
+2. On the second host:
+  1. Allow inbound traffic for `xdp_tests.exe` through Windows Defender Firewall.
+  2. Run `xdp_tests.exe xdp_reflect_test --remote_ip <IP on the first host>`.
```

Comment:
Formatting of these 2 points is not correct. They are coming up as points 3 and 4 (instead of i. and ii.)